android/src/main/java/com/twiliorn/library/TwilioRemotePreviewManager.java

```diff
@@ -70,12 +71,12 @@ protected TwilioRemotePreview createViewInstance(ThemedReactContext reactContext
 
     @Override
     public Map getExportedCustomBubblingEventTypeConstants() {
-      return MapBuilder.builder()
-          .put(
-            ON_FRAME_DIMENSIONS_CHANGED,
-              MapBuilder.of(
-                  "phasedRegistrationNames",
-                  MapBuilder.of("bubbled", ON_FRAME_DIMENSIONS_CHANGED)))
-                  .build();
-  }
-}
+        return MapBuilder.builder()
+                .put(
+                        ON_FRAME_DIMENSIONS_CHANGED,
+                        MapBuilder.of(
+                                "phasedRegistrationNames",
+                                MapBuilder.of("bubbled", ON_FRAME_DIMENSIONS_CHANGED)))
+                .build();
+    }
+}
```

Comment:
Should we add the end of file line just to not have this annoying sing in GH? 🙃  There are a couple of instances.

Reply:
@jagamypriera